### ISSUE TITLE
Chunky won't load selected chunks

### ISSUE THREAD

**Issue by TheOneAlex** [llbit/chunky#357]
**System Specs:**
Macbook Pro
OS X El Capitan 10.11.3
Processor: 2.6 GHz Intel Core i7
Memory: 8 GB 1600 MHz DDR3
Graphics: NVIDIA GeForce GT 650M 1024 MB

I'm using the [World Downloader Mod](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2520465-1-9-1-8-9-world-downloader-mod-create-backups-of) to download my builds off of multiplayer servers. This method has worked in the past, and I have had no issues with chunky until recently.

I'm currently running Minecraft on 1.9, and the World Downloader Mod works fine, too; all it does is download chunks I load in-game to my Minecraft "saves" folder. They load up perfectly fine in Singleplayer.

However, when I select the Singleplayer worlds from my minecraft folder, Chunky won't load the chunks I select:
![640e5f5a144f5819b38333e59ac63622](https://cloud.githubusercontent.com/assets/8887860/14067627/72268c1e-f436-11e5-9cd4-19fede8cf242.png)

Here I am trying it with a different save world:
![93d575336d586257dfd7aa2938ac3f1e](https://cloud.githubusercontent.com/assets/8887860/14067676/ebddcb02-f437-11e5-8fed-686ab08d186c.png)

Chunky freezes at 1 or 2 chunks and won't load any further. I've tried downloading some builds on Minecraft 1.8.9, but I still get the same issue in Chunky. I'm figuring this is a Chunky-related issue as oppose to a problem with my Minecraft saves.

Help!

**Comment by llbit** [comment]
Could you provide a download link for that world, so I can try to reproduce the bug? Thank you!

**Comment by TheOneAlex** [comment]
Here you go:

[Link to save where Chunky won't load chunks
](https://www.sendfilessecurely.com/getfile.aspx?id=bEdGQsaa7uvXvwjlysZDNnS2YTLgmvjiVwD7)

As you've seen in my post on r/Chunky, I'm also having issues with default textures not loading in on a different save file. Hopefully you can provide some insight here as well!
[Link to save where Chunky won't load textures](https://www.sendfilessecurely.com/getfile.aspx?id=bIF38h8Ofif789fxVxc114tBV3sN2T5aeQ79)

**Comment by llbit** [comment]
Thank you for the world file. I was able to verify that this has been fixed in the current development version:

![screenshot 2016-03-30 19 52 03](https://cloud.githubusercontent.com/assets/2757386/14151985/036de04c-f6b1-11e5-8958-ec1423b8fb27.png)

As for the texture loading, that is independent of the world file. You can select a resource pack (texture pack) by going to the Options tab in the main Chunky window (the one with the 2D map).

**Comment by TheOneAlex** [comment]
I'm glad you got the fix for that problem! Any plans on releasing that soon? :3

As for the texture issues, u/jackjt8 suggested I download the default 1.8 textures. I downloaded [these default 1.9 textures instead](http://www.mediafire.com/download/bybjqfaisqcj3tw/1.9+Default+Texture+Pack.zip) and it fixed the issue:

![85516d83b2abaf63aa443f7b3f274733](https://cloud.githubusercontent.com/assets/8887860/14162775/05e2a22a-f6bd-11e5-8d26-c1520a9329e9.png)

I guess the server I downloaded my build from was using _slightly_ customized default textures...
